app/experimenter/experiments/models.py

```diff
@@ -864,6 +877,7 @@ class ExperimentChangeLog(models.Model):
     STATUS_ACCEPTED_LIVE = "Launched Experiment"
     STATUS_LIVE_COMPLETE = "Completed Experiment"
     STATUS_REJECTED = "Rejected Experiment"
+    STATUS_ADDED_RESULTS = "Added Results: "
```

Comment:
Do you need the colon here?

Reply:
ok i took it out... just seemed like if was going to list the sections added that a colon would help... but tif is doing or did a redesign of this area right?